admin/src/css/theme.less

```diff
@@ -42,9 +47,13 @@ html {
 }
 .navigation {
   width: 200px;
+  height: calc(100% - 64px);
```

Comment:
I just tried this and it works in Firefox (`height: 100%` also works) however neither work for me in Chrome! I tried a bunch of options and it looks like the problem is because we use `ng-include` to bring in the navigation template we end up with a useless `<ng-include>` in the DOM which is breaking how the CSS is applied.

The only way to solve this as far as I can tell is by removing the `ng-include` and inlining the navigation template in the index.html directly.

Then you don't need to put `height` on `.navigation` at all...